frontends/concrete-python/concrete/fhe/compilation/module_compiler.py

```diff
@@ -459,7 +452,10 @@ class ModuleCompiler:
     compilation_context: CompilationContext
 
     def __init__(self, functions: List[FunctionDef]):
-        self.configuration = Configuration(composable=True)
+        self.default_configuration = Configuration(
```

Comment:
Maybe we can enable these regardless of the user specified or not instead of having a default configuration.

Reply:
I just wanted to remind the idea, it's okay like this!